### ISSUE TITLE
`getSessionContext` does not work on Vercel / serverless environment

### ISSUE THREAD

**Issue by mattfwood** [blitz-js/blitz#1387]
### What is the problem?
When trying to run `getSessionContext` in `getServerSideProps`, it works as expected locally but not when deployed to Vercel in a serverless environment:

```
2020-10-27T17:13:10.635Z	ERROR	Unhandled error during request: [TypeError: Cannot read property 'sameSite' of undefined]
2020-10-27T17:13:10.635Z	ERROR	[TypeError: Cannot read property 'sameSite' of undefined]
2020-10-27T17:13:10.635Z	ERROR	Unhandled Promise Rejection 	{"stack":"Runtime.UnhandledPromiseRejection: TypeError: Cannot read property 'sameSite' of undefined","message":"TypeError: Cannot read property 'sameSite' of undefined","reason":{"stack":"TypeError: Cannot read property 'sameSite' of undefined","message":"Cannot read property 'sameSite' of undefined"},"promise":{}}
Unknown application error occurred
```

I've tried looking into the Blitz source code to open an MR for this, but i wasn't sure where to start. It looks like it's related to how supertokens is initialized, but i think because it's serverless it can't share that initialized config between lambda functions.

### Steps to Reproduce

1. Create a basic Blitz app
2. Call `getSessionContext` in `getServerSideProps`
```js
export const getServerSideProps = async ({ req, res }) => {
  const session = await getSessionContext(req, res)

  return { props: { userId: session.userId } }
}
```

3. Deploy to Vercel or another serverless platform

### Versions

```
macOS Catalina | darwin-x64 | Node: v12.14.1

blitz: 0.24.3 (global)
blitz: 0.24.1 (local)

  Package manager: yarn 
  System:
    OS: macOS 10.15.6
    CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
    Memory: 324.72 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.14.1 - ~/.nvm/versions/node/v12.14.1/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v12.14.1/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  npmPackages:
    @prisma/cli: 2.8.1 => 2.8.1 
    @prisma/client: 2.8.1 => 2.8.1 
    blitz: 0.24.1 => 0.24.1 
    react: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8 
    react-dom: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8 
    typescript: 4.0.3 => 4.0.3
```

### Other

[Link to a reproduction repo](https://github.com/mattfwood/blitz-auth-error)
[Link to Deployed App](https://blitz-error-reproduction.vercel.app/)

**Comment by flybayer** [comment]
@mattfwood I think this is the same root problem as https://github.com/blitz-js/blitz/issues/794. Can you try adding the lines of code as shown in that issue?

**Comment by mattfwood** [comment]
@flybayer looks like that fixed it! I'll go ahead and close this and let it be resolved in #794
